src/api/upload.ts

```
@@ -2,27 +2,42 @@
 
 import { api } from './http';
 import type { UploadImageResponse } from '@/types/upload';
+import { compressImage, pickUploadPolicy } from '@/utils/image';
 
 export const normalizeImageUrl = (url: string): string => {
   if (!url) return url;
 
   // 앞뒤 공백 및 쌍따옴표 제거
   let cleaned = url.trim().replace(/^"|"$/g, '');
```

ESLint

[error] 'cleaned' is never reassigned. Use 'const' instead.

(prefer-const)